app/src/main/java/net/squanchy/tweets/util/TwitterDateUtils.java

```diff
@@ -0,0 +1,105 @@
+package net.squanchy.tweets.util;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.text.format.DateUtils;
+
+import com.twitter.sdk.android.core.models.Tweet;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+import net.squanchy.R;
+import net.squanchy.support.lang.Optional;
+
+public class TwitterDateUtils {
```

Comment:
Yes, I know, I don't like this code too. This was basically just a copy and paste of `com.twitter.sdk.android.tweetui.TweetDateUtils`. I was already planning of getting read of it as I wrote in the TODO of this PR but if you consider it a blocker I can refactor this code ASAP